lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeAvroDefinitionGenerator.java

```diff
@@ -19,14 +19,20 @@
 
 package org.apache.reef.runtime.multi.client;
 
+import org.apache.reef.annotations.Unstable;
 import org.apache.reef.runtime.multi.utils.avro.MultiRuntimeDefinition;
+import org.apache.reef.tang.annotations.DefaultImplementation;
 
 import java.net.URI;
 
 /**
  * Defines a contract for a multi runtime definition generator.
+ * The multi runtime avro definition contains all the needed information to instantiate
+ * runtimes inside runtime host.
  */
-public interface MultiRuntimeDefinitionGenerator {
+@DefaultImplementation(MultiRuntimeAvroDefinitionGeneratorImpl.class)
+@Unstable
+public interface MultiRuntimeAvroDefinitionGenerator {
```

Comment:
It's convention for the rest of the codebase to precede the name of Avro schemas with the term `Avro`. I think it helps because while I was reading the code, I did not realize that `MultiRuntimeDefinition` was an avro definition before looking into the namespace, which was why I was confused the first time round.